### PR TITLE
add sygnum bank app

### PR DESCRIPTION
**Pull request by fgregoribattista**
[JIRA](https://allianceblockprotocol.atlassian.net/browse/NEXID-1635)

Feature: 
ADD banking Web3 example branded as Sygnum

### CODE REVIEW

**Review by grmkris — APPROVED** [review]

File: root-project/src/features/sygnum-web3/identity/useCheckBankWeb3Compliance.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { api } from "@/utils/api";
+import { useQuery } from "@tanstack/react-query";
+import { useKycBankWeb3Authentication } from "@/features/sygnum-web3/identity/useKycBankWeb3Authenticate";
```
Comment (LizanLycan) [inline root-project/src/features/sygnum-web3/identity/useCheckBankWeb3Compliance.tsx:3]: Try to change all `bankWeb3` names to `signumWeb3`

File: root-project/src/features/sygnum-web3/identity/useCheckBankWeb3Compliance.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+export const useCheckBankWeb3Compliance = (enabled: boolean) => {
+  const { user } = useKycBankWeb3Authentication();
+  const mutation = api.compliance.executeBankWeb3Engine.useMutation();
+
+  return useQuery({
+    queryKey: ["checkBankWeb3Compliance", enabled],
```
Comment (LizanLycan) [inline root-project/src/features/sygnum-web3/identity/useCheckBankWeb3Compliance.tsx:10]: Try to change all `bankWeb3` names to `signumWeb3`

File: root-project/src/features/sygnum-web3/identity/useKycBankWeb3Authenticate.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
+import { useMutation } from "@tanstack/react-query";
+import { create } from "zustand";
+import { createJSONStorage, devtools, persist } from "zustand/middleware";
+import { immer } from "zustand/middleware/immer";
+import { api } from "@/utils/api";
+import { useSignMessage } from "wagmi";
+import { type Address } from "viem";
+import { buildSignatureMessage } from "@nexeraid/identity-sdk";
+
+export const useKycBankWeb3Authentication = () => {
```
Comment (LizanLycan) [inline root-project/src/features/sygnum-web3/identity/useKycBankWeb3Authenticate.tsx:10]: Try to change all `bankWeb3` names to `signumWeb3`

File: root-project/src/features/sygnum-web3/identity/useCheckBankWeb3Compliance.tsx (hunk 1 above)
Comment (fgregoribattista) [reply root-project/src/features/sygnum-web3/identity/useCheckBankWeb3Compliance.tsx]: done!

File: root-project/src/features/sygnum-web3/identity/useKycBankWeb3Authenticate.tsx (hunk 3 above)
Comment (fgregoribattista) [reply root-project/src/features/sygnum-web3/identity/useKycBankWeb3Authenticate.tsx]: done!

File: root-project/src/features/sygnum-web3/identity/useCheckBankWeb3Compliance.tsx (hunk 2 above)
Comment (fgregoribattista) [reply root-project/src/features/sygnum-web3/identity/useCheckBankWeb3Compliance.tsx]: done!

**Review by LizanLycan — APPROVED** [review]

**Pull request merged**